### PR TITLE
Issue/503

### PR DESCRIPTION
**Pull request by henry88lay**
This PR goal is to create a JSON schema as there is no JSON Schema available for pool config file.

### CODE REVIEW

**Comment by Caitlyn-Mills** [comment]
As discussed with @azlam-abdulsalam, please consolidate the files, make sure validation works and add descriptions @henry88lay . Good work, keep it up.

File: src_saleforce_packages/scratchorgpool/sfdx-project.json [hunk 1]
```diff
@@ -1,19 +1,18 @@
...
-    "sfdcLoginUrl": "https://login.salesforce.com",
-    "sourceApiVersion": "51.0",
-    "packageAliases": {
-        "sfpower-scratchorg-pool": "0Ho1P000000blK7SAI",
-        "sfpower-scratchorg-pool@1.0.0-1": "04t1P000000gOkXQAU",
-        "sfpower-scratchorg-pool@1.1.0-1": "04t1P000000gOqzQAE"
+  "packageDirectories": [
+    {
+      "path": "force-app",
+      "default": true,
```
Comment (azlam-abdulsalam) [inline src_saleforce_packages/scratchorgpool/sfdx-project.json:5]: What happened here? there should not be any change in this file

Comment (henry88lay) [reply src_saleforce_packages/scratchorgpool/sfdx-project.json:5]: yeah there shouldn't  be i must of missed this when i was resolving a merge error from my end on this branch i will revert this.

**Review by azlam-abdulsalam — APPROVED** [review]
okie

**Pull request merged**